notebooks/official/migration/UJ5 Vertex SDK AutoML Image Object Detection.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>use latest template.</p><p> </p><p>Will need to update subsequent cells to use BUCKET_URI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>